stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/typing/DefaultTypingUpdatesBuffer.kt

```diff
@@ -35,22 +35,24 @@ import kotlinx.coroutines.launch
  *
  * You should call [clear] before an instance of this class is removed.
  *
+ * @param coroutineScope The coroutine scope used for running the timer and sending updates.
  * @param onTypingStarted Signals that a typing event should be sent.
  * Usually used to make an API call using [io.getstream.chat.android.client.ChatClient.keystroke]
  * @param onTypingStopped Signals that a stop typing event should be sent.
  * Usually used to make an API call using [io.getstream.chat.android.client.ChatClient.stopTyping]
  */
+@Deprecated(
+    message = "This class has been deprecated and will be removed." +
+        "Should you wish to implement your own buffer, you can create " +
+        "a custom implementation of `TypingUpdatesBuffer`.",
```

Comment:
Don't think that markdown is supported here

Reply:
AFAIK it isn't, but I use it to try to signify that it's a piece of code in the AS info bubble.